src/lib/participants/registration.ts

```diff
@@ -1,6 +1,6 @@
-export const registrationOpensAt = +new Date('2023-05-01 00:00:00Z');
-export const eventStartsAt = +new Date('2023-06-30 09:30:00Z');
-export const registrationClosessAt = +new Date('2023-06-26 17:00:00Z');
+export const registrationOpensAt = +new Date('2024-05-01 00:00:00Z');
+export const eventStartsAt = +new Date('2024-06-30 09:30:00Z');
```

Comment:
We don't know this yet 😅

Reply:
its in the future, that is good enough for now